src/content/docs/alerts-applied-intelligence/new-relic-alerts/get-started/your-first-nrql-condition.mdx

```diff
@@ -10,127 +10,278 @@ redirects:
 
-<img
-  title="A screenshot of a NRQL query that finds the average CPU utilization faceted by hostname."
-  alt="A screenshot of a NRQL query that finds the average CPU utilization faceted by hostname."
-  src={accountsNrqlQuery}
-/>
+Our team is creating an alert condition to look for any latency issues in our web transaction time. Now, we're going to create the rules this condition is going to look for. 
 
-<figcaption>
-  This query finds the average CPU utilization in an environment and then breaks it
-  down (facets) by individual host names.
-</figcaption>
+<CollapserGroup>
+  <Collapser
+    id="anomaly-threshold"
+    title="Anomaly threshold (recommended)"
+  >
 
-Once you've created a valid NRQL query, run it. Next, click **Create alert**.
+  Our team is creating this alert condition so that we're notified if web transaction time takes longer than usual. But let's say that we don't care how much longer web transaction time is, we just want to know if transaction time is behaving abnormally. For this specific use case, we'd recommend using our **anomaly threshold**. Our anomaly detection constantly evaluates your data to understand how your system normally behaves. By setting an anomaly threshold you can use our anomaly detection to alert your team if web transaction time deviates from its expected performance. Since you only want to know if web transaction time is behaving unusually, you'd select **upper and lower** because you want notifications of _all_ deviations. But, if you only want alerts if web transaction times takes _longer_ than usual, you'd select **upper only.**
 
-Learn more about the [NRQL syntax](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language/#syntax) to create other useful queries.
+  Next, you need to set the priority level. The priority level determines what will create an incident. We recommend setting your priority level as **critical** for your first alert. You can learn more about priority levels in our [alert condition docs](/docs/alerts-applied-intelligence/new-relic-alerts/advanced-alerts/advanced-techniques/set-thresholds-alert-condition/#threshold-levels).
 
-### Step 2: Name your NRQL condition [#step2-nrql]
+  Next, you will need to define what defines a **critical anomaly threshold violation**. For this first alert, we recommend using our default settings and adjusting to your needs as necessary. So, leave the settings to open an incident "when a query returns a value deviates from the predicted value: **for at least five minutes** by **3 standard deviations**".
```

Comment:
Going on our previous conversations about "you will" (which may just be my paranoia), what do you think about "Next, you must choose what defines"? This eliminates both the "you will" and the double use of "define".

Reply:
Perfect!